main/webapp/modules/core/scripts/views/data-table/data-table-view.js

```diff
@@ -133,8 +138,7 @@ DataTableView.prototype.render = function() {
     this._renderSortingControls(elmts.sortingControls);
   }
 
-  this._renderDataTables(elmts.table[0], elmts.tableHeader[0]);
-  this._div.empty().append(html);
+  this._renderDataTables(elmts.table[0], elmts.headerTable[0]);
```

Comment:
This line throws an error as there is no `headerTable` anymore, it was changed to `tableHeader` in #2719. 

So if I pull the PR now, the projects don't load:

<img width="960" alt="2020-06-17" src="https://user-images.githubusercontent.com/52909743/84870326-cc77a000-b09c-11ea-8051-d03ddd119a08.png">